src/hooks/useAppBootstrap.js

```diff
@@ -90,10 +99,13 @@ export const useAppBootstrap = client => {
       const onboardingParams = parseOnboardingURL(url)
 
       if (onboardingParams) {
-        const { onboardUrl, fqdn } = onboardingParams
+        const { onboardUrl, onboardedRedirection, fqdn } = onboardingParams
```

Comment:
This should be also done on cold boot (here you are in URL interception when app is already running). Cold boot part is in the first useEffect